### DISCUSSION TITLE
Problem with markdown confluence page generation

### DISCUSSION THREAD

**Discussion by TCke83** [bsorrentino/maven-confluence-plugin#247] [Q&A] [answered]
If I use lists in my *.md files it seems to generate wrong formatting in confluence:

example

`# heading`
`* list item 1`
`* list item 2`
`# heading 2`

When this is generated, the confluence page adds the heading to the list, even if I add extra empty lines between the list and the second heading it keeps adding the second heading to the list. Is there a way to stop the list?

**Comment by bsorrentino** [comment, marked as answer]
Hi @TCke83 

I know this bug and I've planned to figure out a solution

in the meanwhile I suggest you a tips to solve such problem. use an **empty blockquote** ( `>` ) as a line separator 

```
# heading
* list item 1
* list item 2
>
# heading 2
```

**Reply by nagper** [reply]
Hi @bsorrentino :

I wonder if it's been solved already?
I am facing the same problem.

Thanks!

**Reply by bsorrentino** [reply]
Hi @nagper 

Currently there isn't a fix available 🥺, take a look to a workaround that I've described above

**Reply by nagper** [reply]
Thanks @bsorrentino,

I do you your workaround, but  there are times when I do not want a newline after a list item.
Hope it'll be solved soon.

Thanks anyway for your work!

**Reply by bsorrentino** [reply]
> but there are times when I do not want a newline after a list item 🧐

Interesting topic I'll go in deep. I'll keep you update

**Reply by nagper** [reply]
Thanks a lot!

**Comment by bsorrentino** [comment]
opened issue #255 to working on fix.

deployed release `7.3-SNAPSHOT` with a candidate solution, please take a chance to test it and let me know

**Reply by nagper** [reply]
7.3-SNAPSHOT works perfectly for me, thanks a lot!!!!
